src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 server.port=8080
+api.version=v1
```

Comment:
With this definition, all apis have the same version, no? But if someday we're changing an api, we only  need to change the version for that particular api, not for all. So maybe it seems uglier, but I believe that each endpoint should have its own version.

Reply:
We just have 1 API with multiple endpoints for my understanding.
Yes, that was the rationale behind my introduction of the api.version variable. Because now we have 2 controllers, both should be accessible at all times by the same api version.
Would be weird if we‘d call one endpoint with v1, the other with v8.